brambling/utils/model_tables.py

```diff
@@ -734,7 +757,7 @@ def person_avoid(self, obj):
 
     def get_eventhousing_csm(self, obj, name):
         eventhousing = obj.get_eventhousing()
-        if eventhousing:
+        if eventhousing and obj.providing_housing:
             return related_objects_list(name)(self, eventhousing)
```

Comment:
Any chance this related_objects_list (and the function in general) could also get replaced while we're in here?